index.js

```diff
@@ -24,7 +361,14 @@ module.exports = {
         'fieldName',
         'required',
         'googleApiKey',
-        'placeholder'
+        'placeholder',
+        'types',
```

Comment:
I think these advanced options should be in a separate "Advanced" field tab.

Reply:
Good idea.